developer-guide.md

```diff
@@ -1,5 +1,9 @@
+
 # Elasticsearch Developer Guide
 
+> ℹ️ You can see a **table of contents** for this guide by clicking the menu icon <img width="28" alt="Screenshot 2023-08-31 at 17 08 56" src="https://github.com/elastic/elasticsearch-labs/assets/32779855/becbabae-798b-409e-ae4d-506beff113ec">
```

Comment:
i wonder where this file exists and will keep remaining? i wonder if this could be an svg instead.